src/main/resources/static/css/forms.css

```diff
@@ -0,0 +1,23 @@
+.login-form {
```

Comment:
Looked into the first commit in this pr and here is the [original style.css](https://github.com/IoT-Ausbildung/grade-calculator/blob/faa9d91537a65bcf4a6d2d435e2f6783bd073d8d/src/main/resources/static/css/style.css)

stuff like for example: _input[type="text"]_ seems to be missing from the new style files. please check that you did not accidentally delete something.

Reply:
login.html doesn't need it anymore since it was used for old login form.